tech.json

```diff
@@ -625,7 +632,10 @@
                         {
                           "name": "canBeetomIceClip",
                           "requires": [ "canNonTrivialIceClip" ],
-                          "note": "Using an attached Beetom to precisely freeze it in place to be used for a clip."
+                          "note": [
+                            "Using an attached Beetom to precisely freeze it in place to be used for a clip.",
+                            "Freezing Beetoms works best when facing right and vertical speed is low."
```

Comment:
assuming this is for freezing attached beetoms?

Reply:
Technically that is what it says 🤓